app/models/manageiq/providers/amazon/storage_manager/ebs/cloud_volume_snapshot.rb

```diff
@@ -61,6 +61,14 @@ def self.create_snapshot(cloud_volume, options = {})
 
   def raw_delete_snapshot
     with_provider_object(&:delete)
+    update!(:status => 'deleting')
+    EmsRefresh.queue_refresh_task(
```

Comment:
so, there is no event for deletion?

Reply:
yep

Reply:
@agrare is this ok? If the event is missing, I think we do not have much choice. (except full refresh)

Reply:
Can we do a targeted refresh of the vm the cloud_volume is connected to?  Not sure what amazon supports here for targets.
Also are you using the task that is returned?  Can you just use `EmsRefresh.queue_refresh`?

Reply:
it's a volume snapshot, so Vm is only optional (volume doesn't have to be attached to anything)

Reply:
Okay sounds like a full is the only way to go then.

Reply:
And why not to just refresh the snapshot like it is now?

Reply:
Oh we can target the snapshot?  If we can do a targeted definitely lets do that.

Reply:
@agrare fixing `queue_refresh`

Reply:
done